### PR TITLE
ROX-24086: Add ACSCS email integration tile & page

### PR DESCRIPTION
**Pull request by alwayshooin**
As it says above. I used the normal email integration table columns/svg as the template for now. Should look like below:
image
image

### CODE REVIEW

**Review by dvail — APPROVED** [review]
Does the feature flag dependency prevent this tile from showing up in on-prem installs?

**Comment by alwayshooin** [comment]
> Does the feature flag dependency prevent this tile from showing up in on-prem installs?

it should prevent the tile from showing up if the feature flags are not turned on. not sure if that answers your question

**Comment by dvail** [comment]
> it should prevent the tile from showing up if the feature flags are not turned on. not sure if that answers your question

I meant that this should only show up on ACSCS right? Looking again though, maybe that is what `'ROX_MANAGED_CENTRAL'` enforces?

File: ui/apps/platform/src/types/featureFlag.ts [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
 export type FeatureFlagEnvVar =
+    | 'ROX_ACSCS_EMAIL_NOTIFIER'
     | 'ROX_ACTIVE_VULN_MGMT'
     | 'ROX_ADMINISTRATION_EVENTS'
     | 'ROX_CLOUD_CREDENTIALS'
     | 'ROX_COMPLIANCE_ENHANCEMENTS'
     | 'ROX_COMPLIANCE_HIERARCHY_CONTROL_DATA'
     | 'ROX_COMPLIANCE_REPORTING'
+    | 'ROX_MANAGED_CENTRAL'
+    | 'ROX_MOVE_INIT_BUNDLES_UI'
```
Comment (dvail) [inline ui/apps/platform/src/types/featureFlag.ts:13]: Is this an intentional return of `ROX_MOVE_INIT_BUNDLES_UI`?

**Review by dvail — APPROVED** [review]
The feature flag change refactor looks good. Only two questions:

- Do we know if anything needs to be done regarding the `useCentralCapabilities()` hook?
- It looks like a stale feature flag was re-added, is this intentional?

File: ui/apps/platform/src/types/featureFlag.ts (hunk 1 above)
Comment (alwayshooin) [reply ui/apps/platform/src/types/featureFlag.ts:13]: oops, definitely not, good catch!

**Comment by alwayshooin** [comment]
> The feature flag change refactor looks good. Only two questions:
> 
> * Do we know if anything needs to be done regarding the `useCentralCapabilities()` hook?
> * It looks like a stale feature flag was re-added, is this intentional?

i messaged johannes about the hook and it turns out we do need it after all. i'll update the PR accordingly and probably re-request another review 😓

**Review by dvail — APPROVED** [review]
Gah, sorry I missed the re-review on this.

Thanks for making the change for the central capabilities, this looks great! 🚢

**Pull request merged**